pkg/distributor/distributor_test.go

```diff
@@ -1291,7 +1292,7 @@ func TestDistributor_Push_LabelRemoval(t *testing.T) {
 			timeseries := ingesters[i].series()
 			assert.Equal(t, 1, len(timeseries))
 			for _, v := range timeseries {
-				assert.Equal(t, tc.expectedSeries, mimirpb.FromLabelAdaptersToLabels(v.Labels))
+				assert.True(t, labels.Equal(tc.expectedSeries, mimirpb.FromLabelAdaptersToLabels(v.Labels)))
```

Comment:
This will be a hell to debug, especially given we're in a loop. `Expected: false, got: true` isn't really meaningful.

Can we create a `test.AssertEqualLabels(t *testing.T, a, b labels.Labels)` in `util/test` that would print a meaningful message if they aren't equal? Similar to what `promtestutil.RequireEqual` prints.

Reply:
sure, I can just do them all as `promtestutil.RequireEqual`

Reply:
Thank you.